index.html

```diff
@@ -1 +1,12 @@
-<div>Hello</div>
\ No newline at end of file
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+</head>
+<body>
+    
```

Comment:
Fix like spacing and indentation.
try using a better syntax extension from VS code